### ISSUE TITLE
Does DTW implementation work with multivariate time series?

### ISSUE THREAD

**Issue by hadware** [slaypni/fastdtw#16]
The question is pretty self explanatory. If it does, maybe it'd be good to mention it in the README.md.

**Comment by brownpanther** [comment]
@hadware  did you figure out if it does?

**Comment by hadware** [comment]
It's been a long time since i've been working on a project that required this, but I think it does.

**Comment by fmeccanici** [comment]
Yes it does this. The numpy array should have the following structure ( I'm using it for Cartesian space trajectories):

        x                      y                  z
[[ 0.41493465 -0.39322514  1.14685164]
 [ 0.41546143 -0.3939345   1.14602891]
 [ 0.41591845 -0.39457089  1.14531319]
 ...
 [ 0.60821661  0.07143943  0.7978909 ]
 [ 0.60813073  0.0714391   0.79794052]
 [ 0.60804489  0.07143876  0.79798989]]

I had to transpose my numpy multidimensional array to acquire this structure.
